Cycle 2408 Changelog (#124)
vatnz-dev/sops: docs/changelog/2024/2408.md

---
  title: Cycle 2408 (08 Aug)
---

--8<-- "includes/abbreviations.md"

This cycle is a minor release, with a small number of changes. Most of these changes come from the renaming of waypoints, under direction from ICAO. 

## SOP Changes

No changes have been made this cycle.

-----

## Data Changes

### Aerodrome

#### NZCH: Christchurch

- The following STARs have been revised:

>     - `BELEE4E`: MSA revised between `VANDA` and `BELEE`

Reviewer: ```suggestion
    - `BELEE 4E`: MSA revised between `VANDA` and `BELEE`
```

  
#### NZDN: Dunedin

- The following approaches have been revised:
    - `ILS RWY 21`: Missed Approach track change

#### NZNR: Napier

- The following waypoints have been renamed:
    - `AKINA` is now `SANON`

- The following procedures have been revised, after hold added at `AROPA`:
    - VOR/DME Y RWY 16
    - VOR/DME Z RWY 16

- VOR/DME RWY 34 has been amended with a new hold at `SANON`

#### NZNS: Nelson

- The following waypoints have been renamed:
    - `QUAIL` is now `ENKIN`

- The following STARs have been amended and re-sequenced:

>     - `LUBSI3C`,`BISEB3A`

Reviewer: ```suggestion
    - `LUBSI 3C`
    - `BISEB 3A`
```

  

- All RNP Approaches have been reviewed.
    - RNP Y RWY 02: IAF `QUAIL` replaced by `ENKIN`

#### NZQN: Queenstown

- The following waypoints have been renamed:
    - `BIKOS` is now `RIBDU`

##### RWY 05

> - The following SIDs have been revised and re-sequenced: 

Reviewer: ```suggestion
- The following SIDs have been revised and re-sequenced with `BIKOS` replaced by `RIBDU`: 
```

>     - `IPNOR5A`,`GIXEL 2`: `BIKOS` replaced by `RIBDU`

Reviewer: ```suggestion
    - `IPNOR 5A`
    - `GIXEL 2`
```

- The following approaches have been revised:
    - RNP Z (AR): Updated minima

##### RWY 23

- The following SIDs have been revised and re-sequenced: 

>     - `ANPOV5B`,`IPNOR5B`: Climb gradients reviewed 

Reviewer: ```suggestion
    - `ANPOV 5B` : `BIKOS` replaced by `RIBDU`, climb gradients reviewed
    - `IPNOR 5B` : `BIKOS` replaced by `RIBDU`, climb gradients reviewed
```

>     - `REDOL3B`,`VALPI 3`

Reviewer: ```suggestion
    - `REDOL 3B`
    - `VALPI 3`
```

- The following approaches have been revised:
    - RNP Z (AR): Updated minima

#### NZTG: Tauranga

- The follwing waypoints have been renamed:
    - `MEZZO` is now `UKMIT`

- The following STARs have been revised and re-sequenced:

>     - `OLDON1D`: Waypoint `MEZZO` replaced by `UKMIT`

Reviewer: ```suggestion
    - `OLDON 1D`: Waypoint `MEZZO` replaced by `UKMIT` (Previously `OLDON 2A`)
```

>     - `OROPI2B`: Waypoint `SALOL` added to STAR.

Reviewer: ```suggestion
    - `OROPI 2B`: Waypoint `SALOL` added to STAR.
```

#### NZFA: Te Hapua Heliport

- The following procedures have been created:
    - `RNP 276`: **CAT H ONLY** approach with IAFs at `VETIR`,`OLTOD`,`KARNI`

>     - `KARNI1P`: new RNP SID, terminates at `KABIL`

Reviewer: ```suggestion
    - `KARNI 1P`: new RNP SID, terminates at `KABIL`
```

#### NZWN: Wellington

- The following waypoints have been renamed:
    - `RIDGE` is now `ATRAB`

> - The following STARs have been revised and re-sequenced:

Reviewer: ```suggestion
```

  
##### RWY 16

> - `TPAPA3D`,`SIMZI3D`,`AVPOD3D`: now CAT `B`,`C`,`D`

Reviewer: ```suggestion
- The following STARs have been reviewed and are now CAT `B`,`C`,`D`
   - `TPAPA 3D`
   - `SIMZI 3D`
   - `AVPOD 3D`
   - `WARDS 4D`: MSA between `PHNIX` and `DORBU` also revised
```

> - `WARDS4D`: now CAT `B`,`C`,`D`, MSA between `PHNIX` and `DORBU` revised

Reviewer: ```suggestion
```

##### RWY 34

> - `SIMZI8A`: Track and MSA between `KRAKA` and `MEEDA` revised

Reviewer: ```suggestion
- The following STARs have been revised:
    - `SIMZI 8A`: Track and MSA between `KRAKA` and `MEEDA` revised
    - `TPAPA 5A`: Track between `BRUDA` and `PEGPU` revised
```

> - `TPAPA5A`: Track between `BRUDA` and `PEGPU` revised

Reviewer: ```suggestion
```

Author: Nice suggestions, @rbywater  Cheers for those, i'll commit them shortly

> - `TPAPA3C`,`SIMZI4C`,`SULRI4C`,`WARDS4C`: now CAT `B`,`C`,`D` and the MSA between `KRAKA` and `MEEDA` revised.

Reviewer: ```suggestion
- The following STARs have been reviewed and are now CAT `B`,`C`,`D`
   - `TPAPA 3C`
   - `SIMZI 4C`
   - `SULRI 4C`
   - `WARDS 4C`: MSA between `KRAKA` and `MEEDA` also revised
```

- The following approaches have been revised:

##### RWY 16

- RNP X (AR): Waypoint `RIDGE` replaced by `ATRAB`

-----

### Standard Routes

No changes have been made this cycle.

> ### Other Dataset Changes

Author: May need to add items here, I don't think I missed anything?
